build-tools/packages/build-infrastructure/package.json

```diff
@@ -14,17 +13,14 @@
 	"type": "module",
 	"exports": {
 		".": {
-			"import": {
+			"default": {
```

Comment:
These changes might not be correct. I can't remember what prompted them. build-infra needs to build both ESM and CJS, because fluid-build is CJS-only at the moment. So we need to include both import and require conditions.

Reply:
Oh, ok... it's the buggy policy handler, `npm-package-exports-field` -- update the fluidBuild.config.cjs file at the repo root to exclude the package.json in build-infra.